### PR TITLE
Ml20

### PR DESCRIPTION
**Pull request by NazarSabara**
Adjusted mapping;
Added user context on UI.

### CODE REVIEW

File: core-service/src/main/java/com/sabara/configuration/ModelMapperConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    private Converter<Set<String>, Set<Group>> groupToEntityConverter = context ->
+        context
+            .getSource()
+            .stream()
+            .filter(StringUtils::isNotBlank)
+            .map(Group::new)
+            .collect(toSet());
+
+    private Converter<Collection<Group>, Set<String>> groupToResourceConverter = context ->
+        context
```
Comment (petrotarakh) [inline core-service/src/main/java/com/sabara/configuration/ModelMapperConfiguration.java:51]: ```context.getSource().stream()``` better to write in single line as a first step of pipeline

File: core-service/src/main/java/com/sabara/model/entity/Group.java [hunk 2]
```diff
@@ -9,11 +12,14 @@
 @Data
 @Entity
 @Table(name = "alliance")
+@RequiredArgsConstructor
```
Comment (petrotarakh) [inline core-service/src/main/java/com/sabara/model/entity/Group.java:15]: ```@RequiredArgsConstructor``` is part of ```@Data```

Comment (NazarSabara) [reply core-service/src/main/java/com/sabara/model/entity/Group.java:15]: Yes, but this does not work when we add `@NoArgsConstructor`

File: core-service/src/main/java/com/sabara/configuration/ModelMapperConfiguration.java (hunk 1 above)
Comment (NazarSabara) [reply core-service/src/main/java/com/sabara/configuration/ModelMapperConfiguration.java]: adjusted

**Pull request merged**